doc/ref_model/chapters/glossary.md

```diff
@@ -53,11 +53,18 @@ To help guide the reader, this glossary provides an introduction to the terminol
 <a name="11.1.2"></a>
 ### 11.1.2 Hardware Layer Terminology
 
-- **Physical Network Function (PNF)**: Implementation of a network function via tightly coupled dedicated hardware and software system. >_*Note:*_ This is a physical NFVI resource with the NF software.
+- **Physical Network Function (PNF)**: Implementation of a network function via tightly coupled dedicated hardware and software system.
+  >_*Note:*_ This is a physical NFVI resource with the NF software.
 - **Hardware resources**: Compute/Storage/Network hardware resources on which the NFVI platform software, virtual machines and containers run on.
 - **NFVI Hardware Profile**: defines the behaviour, capabilities and metrics provided by an NFVI Hardware Layer.
   - **Host Profile**: is another term for a **NFVI hardware profile**.
 - **NFVI Hardware Configuration**: a set of settings (Key:Value) that are applied/mapped to **NFVI** HW deployment.
+- **Simultaneous Multi-threading**: Simultaneous multi-threading (SMT), also known as Hyper-threading in Intel
+CPUs, is a technology that enables the Operating System to see a single core per CPU as two cores per CPU. These new
```

Comment:
@deebhatia The following is confusing: "... a single core per CPU as two cores per CPU ..". Did you mean "... each physical core of a CPU as two virtual cores ...". May I suggest adding "hardware" before "technology as in "... is a hardware technology that enables .." 

"These new cores are called *logical cores* or *threads* .." -- suggest change to "The virtual cores are independent threads of execution."

Reply:
Done. Please check if it is ok.